shopfloor/actions/schema.py

```diff
@@ -109,6 +109,14 @@ def location(self):
             "id": {"required": True, "type": "integer"},
             "name": {"type": "string", "nullable": False, "required": True},
             "barcode": {"type": "string", "nullable": True, "required": False},
+            "progress_info": {
+                "type": "dict",
+                "required": False,
+                "schema": {
+                    "operations_done": {"type": "float", "required": False},
+                    "operations_to_do": {"type": "float", "required": False},
+                },
```

Comment:
```suggestion
            "operation_progress": {
                "type": "dict",
                "required": False,
                "schema": {
                    "done": {"type": "float", "required": False},
                    "todo": {"type": "float", "required": False},
                },
```
not a blocker tho ;)